qemu/tests/cfg/cpu_device_hotpluggable.cfg

```diff
@@ -1,9 +1,11 @@
 # Notes:
 #    For ppc64/ppc64le, please manually specify cpu_model in your environment
 - cpu_device_hotpluggable: install setup image_copy unattended_install.cdrom
-    only ppc64 ppc64le x86_64
+    no aarch64
     Windows:
         no WinXP, WinVista, Win7, Win8, Win10, Win2000, Win2003
```

Comment:
I prefer to use 'only' here to explicitly tell which windows guests support this feature. And when a new win OS comes out, we will check if it supports cpu hotplug and add it accordingly.  If use 'no', when a new non-support os comes out, we may not think about this feature and miss the update needed for this case.

Reply:
If use 'only' here, we need to update the configuration files each time when new win OS comes out, sometimes we miss adding it.
But we can add a new win OS to the `no` list after confirming that the new OS does not support this feature. In this way, the configuration files need to be modified only when the unsupported win OS is came out.

Reply:
We need update cfg later whatever we use. But when a new os is supported, feature owner would definitely know. But if an unsupported new os comes, it may not get feature owner's attention. So I'd perfer 'only'. Thanks.

Reply:
I can accept it because ppc will not test windows guests.
@peixiu It is more relevant to you, what's your suggestion?

Reply:
Hi, 
Our test plan only used your automated one case cpu_device_hotpluggable.hotplug.single_vcpu.with_reboot.shell_reboot, currently both "no" and "only" are ok, but when new windows os coming, it's better default supported here.
For newer windows server os, most probobely will compatible with old version's function before it, this time, the code can be used directly, for newer windows desktop os, may be it also unsupported the function same as currently win10, thus will need update the cfg to except it.
Hope helpful, thanks~

Reply:
We only officially support windows data center version, and won't test other versions for this feature.  It would be much clearer to list supported os version here directly. Thanks.

Reply:
it's ok for me, just when new supported windows os coming, it can be added in supported list  in time is ok~

Reply:
@huangyum Updated cfg based on your preference. And please help update cfg files when new Windows OS comes out.